plugins/outputs/all/all.go

```diff
@@ -16,7 +16,8 @@ import (
 	_ "github.com/influxdata/telegraf/plugins/outputs/kafka"
 	_ "github.com/influxdata/telegraf/plugins/outputs/kinesis"
 	_ "github.com/influxdata/telegraf/plugins/outputs/librato"
-	_ "github.com/influxdata/telegraf/plugins/outputs/mqtt"
+	//_ "github.com/influxdata/telegraf/plugins/outputs/mqtt"
+	_ "github.com/influxdata/telegraf/plugins/outputs/azureTableStorage"
```

Comment:
Let's call is Azure. We won't be adding a separate plugin for every sink.

Reply:
will we be writing code for event hub sink in the same output plugin? I think there should be a different plugin for EH sink as the 2 main functions of output plugin interface are different for table and eh.

Reply:
Let's talk offline.

Reply:
Resolved offline.